templates/search/blocks/featured-search__results-block.html

```diff
@@ -7,10 +7,10 @@ <h2 class="featured-search__heading">{{ bucket.label }}</h2>
             <li class="featured-search__results-list-item">
                 <a href="{% record_url record %}" class="featured-search__results-link">
                     <h3 class="featured-search__results-heading">
-                        {{ record|record_detail:"summaryTitle"|striptags }}
+                        {{ record.summary_title }}
```

Comment:
1) Existing implementation over here has `striptags`
2) noticed some inconsistencies in existing usage of summary title in templates with striptags, Ex: using [striptags](https://github.com/nationalarchives/ds-wagtail/blob/39bf313563744d381acd4c3c8002e20ab42df750/templates/includes/card-group-record-summary-no-image.html#L11) ; without [`striptags` ](https://github.com/nationalarchives/ds-wagtail/blob/39bf313563744d381acd4c3c8002e20ab42df750/templates/includes/records/record-title.html#L6), there should be something in place to get the data cleaned from the api response. Until then could the newly introduced `strip_html` method in this ticket be used to strip out tags from `summary_title` attribute?

Reply:
The API needs to be doing the stripping during ingestion really. But, using `strip_html` should work in the meantime

Reply:
As of 9a88688, `strip_html` is always applied to summary_title values, so we no longer need to use `|striptags` in templates